client/src/main/java/io/confluent/kafka/schemaregistry/utils/QualifiedSubject.java

```diff
@@ -159,6 +159,22 @@ public static String qualifiedContextFor(String tenant, String qualifiedSubject)
     return qs != null ? qs.toQualifiedContext() : "";
   }
 
+  public static String qualifySubjectWithParent(String tenant, String parent, String subject) {
```

Comment:
What does this function do? Does it suppose to return a qualified subject with its parent, first form it in the Default context, if not then form it in parent's context?

Reply:
I get it now, it will return the subject if it's qualified or in the default context, or it will form the qualified subject in parent's context